### PR TITLE
Fixes TypeError when static_labels are defined and metric labels are not

### PR DESCRIPTION
**Pull request by juanjbrown**

### CODE REVIEW

File: prometheus_flask_exporter/__init__.py [hunk 1]
```diff
@@ -480,7 +480,9 @@ def _track(self, metric_type, metric_call, metric_kwargs, name, description, lab
             # merge the default labels and the specific ones for this metric
             combined = dict()
             combined.update(self._static_labels)
-            combined.update(labels)
```
Comment (rycus86) [inline prometheus_flask_exporter/__init__.py]: Could we do this instead?
```
if not labels:
  labels = self._static_labels
else:
  combined...
```

This makes it clear labels is sometimes not there and we don't need the inline comment.

File: tests/test_defaults.py [hunk 2]
```diff
@@ -452,3 +452,43 @@ def test():
             ('version', metrics.version),
             ('app_name', 'Test-App'), ('api_version', 1)
         )
+
+    def test_static_labels_without_metric_labels(self):
```
Comment (rycus86) [inline tests/test_defaults.py]: Awesome! Thanks for adding a test!

File: prometheus_flask_exporter/__init__.py (hunk 1 above)
Comment (juanjbrown) [reply prometheus_flask_exporter/__init__.py]: Yeah, np

**Comment by rycus86** [comment]
Thanks a lot for this again!
I pushed the `0.12.2` release up, just waiting for Travis builds, then it should be available in a few minutes.

**Comment by rycus86** [comment]
See https://pypi.org/project/prometheus-flask-exporter/0.12.2/

**Comment by juanjbrown** [comment]
@rycus86 no problem.  Thanks for the feedback and quick response.

**Pull request merged**